redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -98,7 +98,7 @@ public function log($success, $message)
             ($success ? 'SUCCESS' : 'ERROR'),
             ($this->id ?: '--'),
             $name,
-            strip_tags($message),
+            strip_tags(nl2br($message),'<br><b>'),
```

Comment:
Ähnliches Problem in https://github.com/redaxo/redaxo/issues/1248 - könnte man evtl. eine gemeinsame Vorgehensweise diskutieren und umsetzen.